gm4_disassemblers/data/gm4_disassemblers/generate_disassembly.py

```diff
@@ -0,0 +1,122 @@
+import json
+import urllib.request
+import itertools
+import beet
+
+print("Generating disassembly loot tables...")
+
+diamond_items = ["minecraft:diamond_sword","minecraft:diamond_pickaxe","minecraft:diamond_axe","minecraft:diamond_shovel","minecraft:diamond_hoe","minecraft:diamond_helmet","minecraft:diamond_chestplate","minecraft:diamond_leggings","minecraft:diamond_boots"]
+diamond_durability = [1562, 1562, 1562, 1562, 1562, 364, 529, 496, 430]
+iron_items = ["minecraft:iron_sword","minecraft:iron_pickaxe","minecraft:iron_axe","minecraft:iron_shovel","minecraft:iron_hoe","minecraft:iron_helmet","minecraft:iron_chestplate","minecraft:iron_leggings","minecraft:iron_boots"]
+iron_durability = [251, 251, 251, 251, 251, 166, 241, 226, 196]
+gold_items = ["minecraft:golden_sword","minecraft:golden_pickaxe","minecraft:golden_axe","minecraft:golden_shovel","minecraft:golden_hoe","minecraft:golden_helmet","minecraft:golden_chestplate","minecraft:golden_leggings","minecraft:golden_boots"]
+gold_durability = [33, 33, 33, 33, 33, 78, 113, 106, 92]
+leather_items = ["minecraft:leather_helmet","minecraft:leather_chestplate","minecraft:leather_leggings","minecraft:leather_boots"]
+leather_durability = [56, 81, 76, 66]
+other_items = ["minecraft:turtle_helmet","minecraft:bow","minecraft:crossbow","minecraft:shears","minecraft:fishing_rod","minecraft:flint_and_steel"]
+other_durability = [276, 385, 466, 239, 65, 65]
+normal_items = diamond_items + iron_items + gold_items + leather_items + other_items
+normal_durability = diamond_durability + iron_durability + gold_durability + leather_durability + other_durability
+special_items = ["minecraft:shield"]
+special_durability = [337]
+
+recipes_url = "https://raw.githubusercontent.com/InventivetalentDev/minecraft-assets/1.19.2/data/minecraft/recipes/_all.json"
```

Comment:
Any way to not hardcode the current version would be nice. Also I have a small preference towards <https://raw.githubusercontent.com/misode/mcmeta/1.19.2-summary/data/recipe/data.min.json> :)